lib/3scale_toolbox/commands/service_command/service_list.rb

```diff
@@ -0,0 +1,32 @@
+module ThreeScaleToolbox
+  module Commands
+    module ServiceCommand
+      class ServiceListSubcommand < Cri::CommandRunner
+        include ThreeScaleToolbox::Command
+
+        def self.command
+          Cri::Command.define do
+            name        'list'
+            usage       'list'
+            summary     'service list'
+            description 'List all services'
+            runner ServiceListSubcommand
+
+            param   :source
```

Comment:
Command format is:
`3scale service list [flags] <source>`

Reply:
:+1: